### ISSUE TITLE
Create crawl functionality

### ISSUE THREAD

**Issue by Rafal2228** [Mc01/wgit#3]
Idea:
Instead of forcing the user to manually edit .wgit.json in his home directory, let's create new method which will: 
- take directory as parameter
- find all .git repos in subdirectories
- prompt user for aliases and in case of empty string will skip specific repo
- generate configuration in ~/.wgit folder

Syntax proposal:
`$ wgit crawl $directory`

**Comment by Mc01** [comment]
Good idea would be to ease managing and editing configs created in this way.

Like:
> wgit manage $crawled_session

**Comment by Mc01** [comment]
Additionally to `wgit crawl $root` nice would be to have `wgit add` and `wgit remove`. 
For current directory context.
